props/saber_fett263_buttons.h

```diff
@@ -3919,22 +3919,50 @@ SaberFett263Buttons() : PropBase() {}
   }
 
 #ifdef FETT263_DUAL_MODE_SOUND
-  // Select out.wav based on blade orientation, up = odd, down = even
+  // Select preon wav based on blade orientation if more than 1 file, up = odd, down = even
+  void SelectPreonSound() {
+    Effect* effect;
```

Comment:
These three lines should just be SelectSFXEvenOdd(&SFX_preon);

Reply:
Sorry, should have caught that, fixed.